consumers/campaigner/main.go

```diff
@@ -93,57 +91,25 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) error {
 		limit     int64 = 1000
 	)
 
-	template, err := h.templatesvc.GetTemplate(context.Background(), campaign.TemplateID, msg.UserID)
+	campaignTmp, err := svc.PrepareCampaignTemplateData(h.templatesvc, campaign.TemplateID, msg.UserID)
```

Comment:
U ca nchange the name to be `PrepareTemplateData` since this is in campaigner

Reply:
Shouldn't this method be part of the template service? Also, I think a more appropriate name would be `ParseTemplate` since we are in a way parsing the different parts of the template.